docs/faqs/max_execution_time.rst

```diff
@@ -60,32 +60,32 @@ You can set the maximum session timeout value using the ``max_time`` parameter:
    with Session(max_time="1h"):
        ...
 
-If you don't specify a session ``max_time``, the system defaults are used:
+If you don't specify a session ``max_time``, the system defaults are used.
 
-+--------------+------------------+--------------+-----------+
-| Primitive programs              | Private programs         |
-+==============+==================+==============+===========+
-| Premium User | Open User        | Premium User | Open User |
-+--------------+------------------+--------------+-----------+
-| 8h           | 15m              | 8h           | N/A       |
-+--------------+------------------+--------------+-----------+
+Additionally, there is an *interactive* timeout value. If there are no session jobs queued within that window, the session is temporarily deactivated and normal job selection resumes. During job selection, if the job scheduler gets a new job from the session and its maximum timeout value has not been reached, the session is reactivated until its maximum timeout value is reached. The interactive timeout value is five minutes for premium users and two seconds for open users.
```

Comment:
```suggestion
Additionally, there is an *interactive* timeout value. If there are no session jobs queued within that window, the session is temporarily deactivated and normal job selection resumes. During job selection, if the job scheduler gets a new job from the session and its maximum timeout value has not been reached, the session is reactivated until its maximum timeout value is reached. The interactive timeout value is based on the plan type:
```

So that we only need to change the table in the future.